querystring/index.js

```diff
@@ -22,11 +22,13 @@ function parse (qs) {
       if (index === -1) index = params[i].length
       var key = params[i].substring(0, index)
       var val = params[i].substring(index + 1)
-      if (has(obj, key)) {
-        if (!isArray(obj[key])) obj[key] = [obj[key]]
-        obj[key].push(val)
-      } else {
-        obj[key] = val
+      if (key || val) {
```

Comment:
tbh maybe this should be consistent with the bundled querystring api?

![screen shot 2016-04-08 at 16 58 11](https://cloud.githubusercontent.com/assets/640611/14389580/28379f5c-fdab-11e5-9da5-9b9ae7fe9792.png)